kops/cloud-platforms-test.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: kops/v1alpha2
+kind: Cluster
+metadata:
+  creationTimestamp: null
+spec:
+  api:
+    loadBalancer:
+      type: Public
+  additionalPolicies:
+    node: |
+      [
+        {
+          "Effect": "Allow",
+          "Action": [
+            "route53:ChangeResourceRecordSets"
+          ],
+          "Resource": [
+            "arn:aws:route53:::hostedzone/Z1N3S2UZYKP74S"
```

Comment:
The Route53 hosted zone ID is a per-environment value, so this should also be populated by the `create_cluster_config` script

Reply:
this is a tricky one, and decided to hardcode it to get this working for the sprint.
the problem is that you have to run the terraform outputs for the terraform/global-resources to populate that value, and the terraform variables files is encrypted, making it not work form inside the code-build environment. 
We need to think a way of exporting the key so its possible to de encrypt the repo from within the pipeline. Totally out of scope for this sprint.

Reply:
ok ill take a look